tests/integration/conftest.py

```diff
@@ -20,16 +20,29 @@
 
 import pytest
 
-from cluster_fixture import TrivupFixture, ExternalClusterFixture
+from confluent_kafka.schema_registry.schema import SchemaProvider
```

Comment:
The name conftest.py implies that it is testing the configuration of something, but this file seems to have a cluster factory and an avro schema reader, the former should probably be moved to  cluster_fixture.py and the latter to util.py or similar.